src/App.vue

```diff
@@ -638,27 +618,17 @@ export default class App extends Vue {
         console.log('Launch Darkly update error =', error) // eslint-disable-line no-console
       })
 
-      // fetch the draft filing and resources
+      // handle the filing according to whether we have a business id or temp id
       try {
         if (this.getBusinessId) {
-          await this.fetchBusinessData() // throws on error
-          // this is a Dissolution filing
-          // (only dissolutions have a business id)
-          const resources = await this.handleDraftDissolution()
-          if (!resources) {
-            // go to catch()
-            throw new Error(`Invalid dissolution resources, entity type = ${this.getEntityType}`)
-          }
-          this.setResources(resources)
```

Comment:
Moved into handleDissolutionOrRestoration().

Ditto for handleIaOrRegistration() below.